[BUG]: TypeScript compilation errors with @octokit/rest and latest version of @octokit-types

### What happened?

TypeScript code no longer compiles with the following errors:

```
node_modules/@octokit/plugin-rest-endpoint-methods/dist-types/generated/parameters-and-response-types.d.ts:3091:60 - error TS2339: Property 'POST /enterprises/{enterprise}/{security_product}/{enablement}' does not exist on type 'Endpoints'.

3091             parameters: RequestParameters & Omit<Endpoints["POST /enterprises/{enterprise}/{security_product}/{enablement}"]["parameters"], "baseUrl" | "headers" | "mediaType">;
                                                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@octokit/plugin-rest-endpoint-methods/dist-types/generated/parameters-and-response-types.d.ts:3092:33 - error TS2339: Property 'POST /enterprises/{enterprise}/{security_product}/{enablement}' does not exist on type 'Endpoints'.

3092             response: Endpoints["POST /enterprises/{enterprise}/{security_product}/{enablement}"]["response"];
```

### Versions

`>v19`.

### Relevant log output

```shell
node_modules/@octokit/plugin-rest-endpoint-methods/dist-types/generated/parameters-and-response-types.d.ts:3091:60 - error TS2339: Property 'POST /enterprises/{enterprise}/{security_product}/{enablement}' does not exist on type 'Endpoints'.

3091             parameters: RequestParameters & Omit<Endpoints["POST /enterprises/{enterprise}/{security_product}/{enablement}"]["parameters"], "baseUrl" | "headers" | "mediaType">;
                                                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@octokit/plugin-rest-endpoint-methods/dist-types/generated/parameters-and-response-types.d.ts:3092:33 - error TS2339: Property 'POST /enterprises/{enterprise}/{security_product}/{enablement}' does not exist on type 'Endpoints'.

3092             response: Endpoints["POST /enterprises/{enterprise}/{security_product}/{enablement}"]["response"];
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/package.json
+++ b/package.json
@@ -33,9 +33,9 @@
   "repository": "github:octokit/rest.js",
   "dependencies": {
     "@octokit/core": "^4.1.0",
-    "@octokit/plugin-paginate-rest": "^6.0.0",
+    "@octokit/plugin-paginate-rest": "^6.1.0",
     "@octokit/plugin-request-log": "^1.0.4",
-    "@octokit/plugin-rest-endpoint-methods": "^7.0.0"
+    "@octokit/plugin-rest-endpoint-methods": "^7.1.0"
   },
   "devDependencies": {
     "@octokit/auth": "^3.0.3",

